experimental/traffic-portal/build/package-lock.json

```diff
@@ -1,3207 +1,3207 @@
-			"integrity": "sha512-HjSDRw6gZE5JMggctHBcjVak08+KEVhSIiDzFnT9S9aegmp85S/bReBVTb4QTFaRNptJ9kuYaNhnbNEOkbKb/A=="
-		},
-		"vizion": {
-			"version": "2.2.1",
-			"resolved": "https://registry.npmjs.org/vizion/-/vizion-2.2.1.tgz",
-			"integrity": "sha512-sfAcO2yeSU0CSPFI/DmZp3FsFE9T+8913nv1xWBOyzODv13fwkn6Vl7HqxGpkr9F608M+8SuFId3s+BlZqfXww==",
-			"requires": {
-				"async": "^2.6.3",
-				"git-node-fs": "^1.0.0",
-				"ini": "^1.3.5",
-				"js-git": "^0.7.8"
-			},
-			"dependencies": {
-				"async": {
-					"version": "2.6.4",
-					"resolved": "https://registry.npmjs.org/async/-/async-2.6.4.tgz",
-					"integrity": "sha512-mzo5dfJYwAn29PeiJ0zvwTo04zj8HDJj0Mn8TD7sno7q12prdbnasKJHhkm2c1LgrhlJ0teaea8860oxi51mGA==",
-					"requires": {
-						"lodash": "^4.17.14"
-					}
-				}
-			}
-		},
-		"vm2": {
-			"version": "3.9.18",
-			"resolved": "https://registry.npmjs.org/vm2/-/vm2-3.9.18.tgz",
-			"integrity": "sha512-iM7PchOElv6Uv6Q+0Hq7dcgDtWWT6SizYqVcvol+1WQc+E9HlgTCnPozbQNSP3yDV9oXHQOEQu530w2q/BCVZg==",
-			"requires": {
-				"acorn": "^8.7.0",
-				"acorn-walk": "^8.2.0"
-			}
-		},
-		"word-wrap": {
-			"version": "1.2.4",
-			"resolved": "https://registry.npmjs.org/word-wrap/-/word-wrap-1.2.4.tgz",
-			"integrity": "sha512-2V81OA4ugVo5pRo46hAoD2ivUJx8jXmWXfUkY4KFNw0hEptvN0QfH3K4nHiwzGeKl5rFKedV48QVoqYavy4YpA=="
-		},
-		"wrappy": {
-			"version": "1.0.2",
-			"resolved": "https://registry.npmjs.org/wrappy/-/wrappy-1.0.2.tgz",
-			"integrity": "sha512-l4Sp/DRseor9wL6EvV2+TuQn63dMkPjZ/sp9XkghTEbV9KlPS1xUsZ3u7/IQO4wxtcFB4bgpQPRcR3QCvezPcQ=="
-		},
-		"ws": {
-			"version": "7.4.6",
-			"resolved": "https://registry.npmjs.org/ws/-/ws-7.4.6.tgz",
-			"integrity": "sha512-YmhHDO4MzaDLB+M9ym/mDA5z0naX8j7SIlT8f8z+I0VtzsRbekxEutHSme7NPS2qE8StCYQNUnfWdXta/Yu85A==",
-			"requires": {}
-		},
-		"xregexp": {
-			"version": "2.0.0",
-			"resolved": "https://registry.npmjs.org/xregexp/-/xregexp-2.0.0.tgz",
-			"integrity": "sha512-xl/50/Cf32VsGq/1R8jJE5ajH1yMCQkpmoS10QbFZWl2Oor4H0Me64Pu2yxvsRWK3m6soJbmGfzSR7BYmDcWAA=="
-		},
-		"yallist": {
-			"version": "3.1.1",
-			"resolved": "https://registry.npmjs.org/yallist/-/yallist-3.1.1.tgz",
-			"integrity": "sha512-a4UGQaWPH59mOXUYnAG2ewncQS4i4F43Tv3JoAM+s2VDAmS9NsK8GpDMLrCHPksFT7h3K6TOoUNn2pb7RoXx4g=="
-		},
-		"yamljs": {
-			"version": "0.3.0",
-			"resolved": "https://registry.npmjs.org/yamljs/-/yamljs-0.3.0.tgz",
-			"integrity": "sha512-C/FsVVhht4iPQYXOInoxUM/1ELSf9EsgKH34FofQOp6hwCPrW4vG4w5++TED3xRUo8gD7l0P1J1dLlDYzODsTQ==",
-			"requires": {
-				"argparse": "^1.0.7",
-				"glob": "^7.0.5"
-			}
-		}
-	}
+  "name": "traffic-portal-server",
+  "version": "1.0.0",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "traffic-portal-server",
+      "version": "1.0.0",
+      "license": "Apache-2.0",
+      "dependencies": {
+        "pm2": "^5.3.1"
+      },
+      "engines": {
+        "node": ">=20.11.0"
```

Comment:
One last place to change to 20.0.0

Reply:
Fixed in https://github.com/apache/trafficcontrol/pull/8012